website/docs/r/express_route_connection.html.markdown

```diff
@@ -96,6 +96,8 @@ The following arguments are supported:
 
 * `enable_internet_security` - (Optional) Is Internet security enabled for this Express Route Connection?
 
+* `enable_private_link_fast_path` - (Optional) Bypass the ExpressRoute gateway when accessing private-links. ExpressRoute FastPath (`express_route_gateway_bypass_enabled`) must be enabled.
```

Comment:
What do you mean by 
> ExpressRoute FastPath must be enabled?

Reply:
The other option `express_route_gateway_bypass_enabled` must be set to `true` for the `private_link_fast_path_enabled` to be successful, both options must be set to `true` according to azure docs: https://learn.microsoft.com/en-us/azure/templates/microsoft.network/expressroutegateways?pivots=deployment-language-terraform#expressrouteconnectionproperties-2

Reply:
In that case can we format the property description for consistency
```
* `enable_private_link_fast_path` - (Optional) Bypass the Express Route gateway when accessing private-links. When enabled `express_route_gateway_bypass_enabled` must be set to `true`. Defaults to `false`.
```

In addition we should probably add some validation within the create for this case?

Reply:
Good catch with the validation, that is now being implemented. I've also updated the documentation as you proposed.